CHANGELOG.md

```diff
@@ -1,5 +1,90 @@
 # Changelog
 
+## [v1.4.0-rc](https://github.com/googleforgames/agones/tree/v1.4.0-rc) (2020-02-26)
+
+[Full Changelog](https://github.com/googleforgames/agones/compare/v1.3.0...v1.4.0-rc)
+
+**Breaking changes:**
+
+- Update documentation \(examples and website\) to use Kubernetes 1.14 [\#1364](https://github.com/googleforgames/agones/pull/1364) ([roberthbailey](https://github.com/roberthbailey))
+- Update terraform configs to use Kubernetes 1.14 [\#1342](https://github.com/googleforgames/agones/pull/1342) ([roberthbailey](https://github.com/roberthbailey))
+- Update developer tooling to Kubernetes 1.14 [\#1341](https://github.com/googleforgames/agones/pull/1341) ([roberthbailey](https://github.com/roberthbailey))
+- Change the GameServerAllocationPolicy version to stable. [\#1332](https://github.com/googleforgames/agones/pull/1332) ([pooneh-m](https://github.com/pooneh-m))
+- Changing the allocator API to gRPC [\#1314](https://github.com/googleforgames/agones/pull/1314) ([pooneh-m](https://github.com/pooneh-m))
+
+**Implemented enhancements:**
+
+- Change GameServerAllocationPolicy CRD version to stable [\#1290](https://github.com/googleforgames/agones/issues/1290)
```

Comment:
This perhaps should not be here? it is mentioned in the Breaking change section.

Reply:
This perhaps should not be here? it is mentioned in the Breaking change section. :+1:

Reply:
I added the `breaking` tag to the issue, so it will move in the Changelog when we regen for the stable release. :+1: